tests/rptest/tests/partition_balancer_test.py

```diff
@@ -475,16 +475,27 @@ def test_full_nodes(self):
         self.topic = TopicSpec(partition_count=32)
         self.client().create_topic(self.topic)
 
-        # produce around 2GB of data, this should be enough to fill node disks
-        # to a bit more than 70% usage on average
-        msg_count = 19_000
+        used_space_before_produce_bytes = int(
+            self.redpanda.nodes[0].account.ssh_output(
+                "df --output=used /var/lib/redpanda").decode("utf-8").split(
+                    "\n")[1]) * 1024
+
+        # Need to fill disk more than 70% usage
+        # 0.8 - additional data for segments coefficient
```

Comment:
Does this coefficient reflect the fact that data on disk takes more space than produced data? If so, would benefit from a comment discussing sources of a discrepancy (batch headers, fallocated space, etc.) and how it was determined.